src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/resendtogaps/ResendToGapsMessageHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.resendtogaps;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.openMocks;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.HearingRoute.LIST_ASSIST;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.HearingState.CANCEL_HEARING;
+
+import junitparams.JUnitParamsRunner;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import uk.gov.hmcts.reform.sscs.model.hearings.HearingRequest;
+import uk.gov.hmcts.reform.sscs.model.servicebus.SessionAwareServiceBusMessagingService;
+import uk.gov.hmcts.reform.sscs.service.servicebus.HearingMessagingServiceFactory;
+
+@RunWith(JUnitParamsRunner.class)
```

Comment:
You can use MockitoJUnitRunner here instead then put @InjectMocks onto the handler. You won't need to openMocks with this

Reply:
I just tried.. This doesn't work because of the feature flag on the constructor, need to create it manually.